tools/localazy/config.json

```diff
@@ -88,7 +88,8 @@
       "includeRegex": [
         "screen_room_details_.*",
         "screen_room_member_list_.*",
-        "screen_dm_details_.*"
+        "screen_dm_details_.*",
+        "screen_create_room_.*"
```

Comment:
Yeah, this confused me a bit as well. I guess we could duplicate the strings and hopefully localazy would do magic to mean they don't need to be translated twice? :thinking:

Reply:
We can import the same strings into several modules, but that is not ideal.
I would create new keys on localazy with new prefix for the room details edit (form) screen, to avoid confusion).